lang/c++/build.sh

```diff
@@ -71,7 +71,6 @@ function do_dist() {
   fi
 }
 
-(mkdir -p build; cd build; cmake --version; cmake -G "Unix Makefiles" ..)
```

Comment:
The `clean` target fails if another target that creates the `build/` folder is not executed first.
```
avro/lang/c++ on  mkmkme/no-make [$?] via △ v3.22.1 
❯ ./build.sh clean
Error: /home/martin/git/apache/avro/lang/c++/build is not a directory
```

Reply:
Ah true! Thanks, I'll fix it.

Reply:
Added a configuration step before the loop